ansible/vars/icds/icds_public.yml

```diff
@@ -175,7 +175,7 @@ etc_host_lines:
   - '10.247.24.13		www.icds-cas.gov.in'
 etc_host_lines_removed: ['etc_host_lines']
 
-http_proxy_address: '10.247.63.132'
+http_proxy_address: '10.247.24.16'
```

Comment:
no problem, it just uses itself as a proxy, it's part of the local network allow to use it. sames goes for the patch server, it can use itself as a proxy, it gains the benefits of cached files.